src/Spryker/Zed/EventBehavior/Dependency/Plugin/EventResourceBulkRepositoryPluginInterface.php

```diff
@@ -17,8 +17,9 @@ interface EventResourceBulkRepositoryPluginInterface extends EventResourcePlugin
      *
      * @param int $offset
      * @param int $limit
+     * @param array<int> $ids
      *
      * @return array<\Spryker\Shared\Kernel\Transfer\AbstractTransfer>
      */
-    public function getData(int $offset, int $limit): array;
```

Comment:
here needs to adjust new interface. will be better to have the transfer here @ehsanmx